_studio/mfx_lib/decode/av1/src/mfx_av1_dec_decode.cpp

```diff
@@ -958,6 +958,10 @@ mfxStatus VideoDECODEAV1::SubmitFrame(mfxBitstream* bs, mfxFrameSurface1* surfac
                 sts = MFX_ERR_MORE_DATA;
             }
 
+            if (umcRes == UMC::UMC_ERR_INVALID_PARAMS)
```

Comment:
I think we just need to add handling of unknown error here. And we need to figure out why we get unknown error from GetFrame call. Basically such unknown errors from driver should have been mapped to device failed error.

Reply:
Unhandled driver errors are mapped to MFX_ERR_DEVICE_FAILED. And rest of the errors are mapped to MFX_ERR_UNKNOWN.